src/styles/header.scss

```diff
@@ -0,0 +1,67 @@
+@use './variables.scss' as *;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  width: auto;
+  background-color: #fff;
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+  &__nav {
+    display: block;
+  }
+}
+
+.list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  &__link {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    margin-left: 20px;
```

Comment:
The margin-left is set to 20px for each list__link. This could cause alignment issues on smaller screens. Consider using a responsive design approach to adjust margins based on screen size.